### DISCUSSION TITLE
Flow Chart & Notes - Filtering / Posting

### DISCUSSION THREAD

**Discussion by BrysonMG** [nss-day-cohort-47/C47-Discussions#54] [Class Notes and Discussion]
Today I made a pretty thorough flow chart that includes things like:

- Pulling from JSON file
- Converting arrays and objects to HTML and getting them on the DOM
- Filtering data and updating the DOM to reflect it
- Posting to the JSON file with data collected by a form

This chart is pretty big, but when I zoom in on my copy of it, everything is pretty clear. Let me know if there's resolution issues for you and I'll try and find a fix for it.

Below is the image itself, but here is the image address as an alternative way to view it:
https://s3.amazonaws.com/assets.mockflow.com/app/wireframepro/company/C842829a29ee8dbd6b40c4280606b1d31/projects/Mdf9d825009f6f23aa3b93f02b2f26f0d1615316814293/pages/D1cfc5cd4d61cfca7026509af8a039ba5/image/D1cfc5cd4d61cfca7026509af8a039ba5.png?1615498240233

![Complete](https://user-images.githubusercontent.com/59521212/110857796-e25a7000-827e-11eb-89e8-10e12b95c030.png)

**Comment by DavidEdwardDarden** [comment]
Looks great. Thanks for posting.

**Reply by DavidEdwardDarden** [reply]
There are some places where your "explaining text" overlaps.... like beside the... "let allEntries = [] " block

**Reply by BrysonMG** [reply]
Yeah I caught that right after I posted it, I fixed two spots it was doing it, let me know if you see any others. If there's another spot doing it, I'm overlooking it.

**Comment by BryanNilsen** [comment]
@BrysonMG this is fantastic work and I'm sure really helpful in following the flow through the code!! I LOVE THIS!!

**Reply by BrysonMG** [reply]
It definitely made me stop and really think about what's going on in order to actually put this together. Looking at the chart as a whole is much easier to follow than just reading through walls of code across multiples tabs.

**Reply by Chadwell-Clark** [reply]
Very helpful. I tried tonight to go through some projects and add comments about what every module is doing and this helped me reinforce what we are learning. Thanks for sharing with the class. Keep em coming!
